src/SFA.DAS.Forecasting.Jobs.Application.UnitTests/Handlers/WhenHandlingEmployerLevyRefreshComplete.cs

```diff
@@ -41,7 +42,7 @@ public async Task If_No_Levy_Calculated_Should_Not_Trigger_Forecast()
             await _sut.Handle(_event);
 
             // Assert
-            _levyForecastServiceMock.Verify(mock => mock.TriggerLevyForecast(It.IsAny<short>(), It.IsAny<string>(), It.IsAny<long>()), Times.Never);
+            _levyForecastServiceMock.Verify(mock => mock.TriggerLevyForecast(_event.PeriodMonth, _event.PeriodYear, _event.AccountId), Times.Never);
```

Comment:
IsAny() is OK (and prefered) for the Times.Never case.  It's just the Times.Once that should be an exact match.

Reply:
You are right, I did a blanket change on those tests without thinking about it